packages/dai-plugin-governance/contracts/addresses/testnet.json

```diff
@@ -1,9 +1,9 @@
 {
   "POLLING": "0x9A2810a12197CB844af4BA74a4b81bC1CA727130",
   "BATCH_POLLING": "0xa9D33Ce18803b0742460ffb1b33B6c40f95178BC",
-  "VOTE_PROXY_FACTORY": "0x1F398c945F8de27650c74C1881f1c96eC8720472",
-  "MCD_ESM": "0xb379BdC949F09e0E1c866785D1896B763E1387b0",
-  "MCD_END": "0x31880D3A1E5cbF402b6F9840e7aDC6E7962b771B",
+  "VOTE_PROXY_FACTORY": "0x9fb9B0CCb0b6bcA8ff7667c1c1733F4469e27Af1",
+  "MCD_ESM": "0x4A504460C1ea6c1945D07619B546e4629fd2A5dB",
+  "MCD_END": "0x3F35940dD9f42F7560fa08F506c81d99fed870a7",
```

Comment:
@b-pmcg do you know if these should be changed here?

Reply:
These all got new deployments because I updated the dss-deployment-scripts in the testchain